### PR TITLE
added header and hero section

### PR DESCRIPTION
**Pull request by AYUSH-TRIVEDI6936484**

### CODE REVIEW

File: app/components/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          delay: 2500,
+          disableOnInteraction: false,
+        }}
+        pagination={{ clickable: true }}
+        loop={true}
+        modules={[Autoplay, Pagination]}
+        className="max-w-[-webkit-fill-available]"
+        style={{ height: '100vh' }}
+      >
+        {homePageData.storeBanners.map((banner) => (
```
Comment (Himanshu7614) [inline app/components/Hero.tsx]: use banner model like this ({banner}:{banner:BannerModelName})

File: app/components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  //   return () => {
+  //     window.removeEventListener('scroll', handleScroll);
+  //   };
+  // }, []);
+
+  return (
+    <div className={`fixed top-0 left-0 w-full flex justify-center items-center z-50 transition-all duration-300 h-[74px] bg-[#fce9d1]`}>
+      <nav className='max-w-[1200px] 2xl:max-w-[1440px] mx-auto flex justify-center w-full items-center'>
+        <div className="flex justify-between items-center w-full">
+          {/* Logo Container */}
```
Comment (Himanshu7614) [inline app/components/Navbar.tsx]: remove this type of comments

File: app/components/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            <a href="#" className="relative flex text-[#111111] hover:text-[#e4b478]">
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                 <path d="M7 8V6C7 4.67392 7.52678 3.40215 8.46447 2.46447C9.40215 1.52678 10.6739 1 12 1C13.3261 1 14.5979 1.52678 15.5355 2.46447C16.4732 3.40215 17 4.67392 17 6V8H20C20.2652 8 20.5196 8.10536 20.7071 8.29289C20.8946 8.48043 21 8.73478 21 9V21C21 21.2652 20.8946 21.5196 20.7071 21.7071C20.5196 21.8946 20.2652 22 20 22H4C3.73478 22 3.48043 21.8946 3.29289 21.7071C3.10536 21.5196 3 21.2652 3 21V9C3 8.73478 3.10536 8.48043 3.29289 8.29289C3.48043 8.10536 3.73478 8 4 8H7ZM7 10H5V20H19V10H17V12H15V10H9V12H7V10ZM9 8H15V6C15 5.20435 14.6839 4.44129 14.1213 3.87868C13.5587 3.31607 12.7956 3 12 3C11.2044 3 10.4413 3.31607 9.87868 3.87868C9.31607 4.44129 9 5.20435 9 6V8Z" fill="#111111"></path>
+             </svg>
+                 </a>
+                 {/* <span className="absolute block top-[1.5rem] right-[10rem] h-[16px] min-w-[16px] text-[10px] font-bold bg-[#111111] text-[#ffffff] leading-[16px] text-center p-[0 0.3em] rounded-[50%]">
```
Comment (Himanshu7614) [inline app/components/Navbar.tsx]: remove this unused line

File: app/components/Hero.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
+'use client';
+import Image from "next/image";
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/css'; 
+import 'swiper/css/pagination'; 
+import { Autoplay, Pagination } from 'swiper/modules';
+import { BannerImage } from 'D:/intern-ecommerce-project/models/banner';
+
+export const homePageData = {
```
Comment (Himanshu7614) [inline app/components/Hero.tsx]: import data from data page don't create another json data

**Review by Himanshu7614 — CHANGES_REQUESTED** [review]
check comments

File: app/components/Hero.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+import Image from 'next/image';
+import {Swiper, SwiperSlide} from 'swiper/react';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import {Autoplay, Pagination} from 'swiper/modules';
+import { homePageData } from '@/data/homepage';
+import { BannerImage } from '@/models/banner';
+
+export default function Hero() {
+  return (
```
Comment (Himanshu7614) [inline app/components/Hero.tsx]: How do I iterate through banner data?

File: next.config.js [hunk 6]
```diff
@@ -12,6 +12,7 @@ const nextConfig = {
     imageSizes: [
       16, 32, 48, 64, 96, 128, 196, 256, 300, 392, 512, 640, 750, 828, 1080,
     ],
+    domains: ['d3fnp5f2rq7t40.cloudfront.net'],
```
Comment (Himanshu7614) [inline next.config.js]: remove this line

Comment (AYUSH-TRIVEDI6936484) [reply next.config.js]: Done

File: app/components/Navbar.tsx [hunk 7]
```diff
@@ -0,0 +1,131 @@
+'use client';
```
Comment (Himanshu7614) [inline app/components/Navbar.tsx]: change Navbar to header.tsx

**Review by Himanshu7614 — CHANGES_REQUESTED** [review]
move component file from app to root folder

**Pull request merged**